zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -441,6 +443,12 @@ def get_cluster_services(
         service_namespace = obj['metadata']['namespace']
         labels = obj['metadata'].get('labels', {})
         version = labels.get('version', '')
+        initialize_key = config_endpoints_map.get(service.name, {}).get('initialize', {})
+
+        if len(initialize_key) >= 10 and initialize_key.isdigit():
```

Comment:
I'm afraid `len > 10` is not exactly correct. In general it's uint64, and since it could be either missing, be an empty string or an integer, the last check `isdigit` is probably sufficient.